README.md

```diff
@@ -105,8 +105,8 @@ Please cite the following paper if you use ensmallen in your research and/or
 software. Citations are useful for the continued development and maintenance of
 the library.
 
-* Ryan R. Curtin, Marcus Edel, Rahul Ganesh Prabhu, Suryoday Basak, Zhihao Lou, Conrad Sanderson.  
-  [The ensmallen library for flexible numerical optimization](https://jmlr.org/papers/volume22/20-416/20-416.pdf).  
+* Ryan R. Curtin, Marcus Edel, Rahul Ganesh Prabhu, Suryoday Basak, Zhihao Lou, Conrad Sanderson.
```

Comment:
Please keep the two blank spaces at the end of each line containing the author names and article name. Markdown interprets two spaces followed by a linebreak as a "forced linebreak".  In this case we need the forced linebreaks to ensure proper layout (to facilitate reading).